client/python/samples/README.md

```diff
@@ -0,0 +1,230 @@
+# OpenVINO&trade; Model Server Client Library Example Clients
+
+This document contains examples to run *GetModelStatus*, *GetModelMetadata*, *Predict* functions over gRPC API.
+
+It covers following topics:
+* <a href="#grpc-api">gRPC API Example Clients </a>
+* <a href="#model-status">Model status </a>
+* <a href="#model-metadata">Model metadata </a>
+* <a href="#predict-numeric">Predict numeric format </a>
+* <a href="#predict-binary">Predict binary format </a>
+* <a href="#detect-vehicle">Detect vehicle </a>
```

Comment:
Actually can we name those points by the name of the scripts?
* get_grpc_model_status.py
* get_grpc_metadata.py
* resnet_grpc_predict.py
* resnet_grpc_predict_binary.py
* vehicle_detection_predict_binary.py